packages/kbn-optimizer/limits.yml

```diff
@@ -117,7 +117,7 @@ pageLoadAssetSize:
   securitySolution: 66738
   serverless: 16573
   serverlessObservability: 30000
-  serverlessSearch: 26000
+  serverlessSearch: 30000
```

Comment:
I need to understand how this changes since the commit that increased the bundle size was me rebasing with main and removing a file 🤨

Reply:
If you're interested in digging into the bundle the label `ci:build-webpack-bundle-analyzer` will generate an html report for you.